naptime-graphql/src/main/scala/org/coursera/naptime/ari/graphql/schema/exceptions.scala

```diff
@@ -1,4 +1,8 @@
 package org.coursera.naptime.ari.graphql.schema
 
+import sangria.execution.UserFacingError
+
 case class SchemaGenerationException(msg: String) extends Exception(msg)
 case class SchemaExecutionException(msg: String) extends Exception(msg)
+
+case class NotFoundException(msg: String) extends Exception(msg) with UserFacingError
```

Comment:
Can this be documented as to what not-found corresponds to? Does it correspond to resource not found? Or element not found? Or something else? Alternatively, rename to something less ambiguous. (i.e. `ElementNotFoundException`.)